sale_cancel_remaining/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module allows you to cancel the remaining quantity on sale order by adding
+a dedicated action to sale lines. It also add two new fields to track canceled
+and remaining to deliver quantities.
```

Comment:
You should add:

```
This module differs from the original odoo behavior in the following way:

* In odoo, if the update of the qty ordered is allowed on the sale order even 
  when the order is confirmed, odoo will recompute the required stock operations 
  according to the changes done to the qty ordered. This change is possible
  even the stock operations are started for this sale order line.
* In this module, the qty ordered is not updated on the sale order line to keep
  the original qty ordered. At the same time, we cancel only the stock moves
  for the remaining qty to deliver. This is only possible if no operation is
  started for this sale order line.


.. warning::
    
    It's not recommended to use this module at the same time that allowing
    the update of the qty ordered on the sale order line after the order is
    confirmed. This could lead to unpredictable behavior.

```